io.github.cges30901.hmtimer.json

```diff
@@ -0,0 +1,74 @@
+{
+  "app-id": "io.github.cges30901.hmtimer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "hmtimer",
+  "finish-args": [ 
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=~/.config/autostart",
+    "--share=network",
+    "--system-talk-name=org.freedesktop.login1",
+    "--socket=pulseaudio"
+  ],
+  "cleanup": ["/include", "*.a"],
+  "modules": [
+    {
+      "name": "hmtimer",
+      "buildsystem": "simple",
+      "build-commands": [
+            "qmake DEFINES+=FLATPAK",
```

Comment:
Use `"buildsystem": "qmake"`. You can then pass `DEFINES+=FLATPAK` to `"config-opts"`.

The rest of the install commands can go in a `"post-install"` step.